apps/spack/package.py

```diff
@@ -31,7 +31,7 @@ class Hpcx(AutotoolsPackage):
     provides('mpi')
                                                                                                    
     def setup_dependent_environment(self, spack_env, run_env, dependent_spec):
-        self.prefix.bin = "/opt/hpcx-v2.5.0-gcc-MLNX_OFED_LINUX-4.7-1.0.0.1-redhat7.7-x86_64/ompi/bin"
+        self.prefix.bin = "/opt/hpcx-v2.6.0-gcc-MLNX_OFED_LINUX-5.0-1.0.0.0-redhat7.7-x86_64/ompi/bin"
```

Comment:
Can this be dynamically discovered based on the current HPC image you are running on ? We probably don't want to change this file each time the image is upgraded.

Reply:
ok thanks, I have updated package.py to get the path from the module file.